Source/LibationWinForms/grid/GridEntry.cs

```diff
@@ -1,101 +1,81 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Drawing;
-using System.Linq;
-using ApplicationServices;
-using DataLayer;
+﻿using DataLayer;
 using Dinah.Core.DataBinding;
-using Dinah.Core;
 using Dinah.Core.Drawing;
 using LibationFileManager;
-using System.Threading.Tasks;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;using System.Drawing;
+using System.Linq;
 
 namespace LibationWinForms
 {
-	/// <summary>
-	/// The View Model for a LibraryBook
-	/// </summary>
-	internal class GridEntry : AsyncNotifyPropertyChanged, IMemberComparable
+	public interface IHierarchical<T> where T : class
 	{
-		#region implementation properties NOT exposed to the view
-		// hide from public fields from Data Source GUI with [Browsable(false)]
+		T Parent { get; }
```

Comment:
I was hoping to diligently pour through the changes in this class but I gotta be honest, there's too much changed. It'll be easier to pull it locally.

Reply:
Understood. Moving SortableBindingList into libation and making it non-generic will allow for cleaner code, and this interface can go away.

Reply:
Let me know of anything you want moved from dinah.core.databinding => libation and I'll kill it in dinah.core. It's only being used in Libation.

Reply:
Keep everything in Dinah for now.  It;'s still used by RemobeBooksDialog.  We can migrate that DataGridView to the new binding later, but quite frankly I don't want to deal with that right now.